daemon/inspect.go

```diff
@@ -117,6 +118,10 @@ func (daemon *Daemon) getInspectData(container *container.Container, size bool)
 		hostConfig.LogConfig.Config = daemon.defaultLogConfig.Config
 	}
 
+	if container.Name[0] == '/' {
```

Comment:
the "if" may not even be needed; `strings.TrimLeft()` will only strip the `/` if there is one.
I wonder if we should change `container.Name` here, or do it at https://github.com/docker/docker/pull/18524/files#diff-9d7e088e8ca8f96f2ac0af044001ae04R147, because we do formatting for other fields there as well